src/lib/storage/value_segment.cpp

```diff
@@ -34,14 +34,13 @@ size_t ValueSegment<T>::size() const {
 }
 
 template <typename T>
-const std::vector<T>& ValueSegment<T>::values() const {
-  return _values;
+size_t ValueSegment<T>::estimate_memory_usage() const {
+  return size() * sizeof(T);
```

Comment:
Most other files define one liner functions in the same line. It would make sense to change it either in this file or the others for consistency sake

Reply:
what about strings?

Reply:
Well, if you know how to solve it, let us know :D

Reply:
"Strings may use additional heap storage outside of the std::stringobject; you do not need to take that into account."

Reply:
I will let the linter decide how he likes it best